forms/CommentCreateForm.php

```diff
@@ -67,9 +70,11 @@ public function save()
         $Comment = $this->Comment;
 
         if (empty($this->id)) {
-            $Comment = new Comments\models\Comment();
+            $Comment =  \Yii::$app->getModule(Comments\Module::NAME)->model('comment');
```

Comment:
Yeah, no problem using the DI instead of the manual mode, but, do you consider appropriate to add the model mapping in the config of the module? 

I think that would be a good idea to simplify the rewriting of the classes and will make less verbose doing that instead of using the `$container->set(....` 

Only problem would be that we will still have to call the module for that, because the models will be injected in the model init time, and it isn't called untill you reference it the first time, given it isn0t a bootstrap module. But that you already use variables from the module, that won't be such a big deal, only that you have to load it at the start of the widgets.